### PR TITLE
Nullable annotations

### PR DESCRIPTION
**Pull request by davkean**
Latest compiler version introduced new warnings, clear up the warnings.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/InputNodeGraphActionHandlerBase.cs [hunk 1]
```diff
@@ -70,7 +70,7 @@ public sealed override bool TryHandleRequest(IGraphContext graphContext)
                 }
 
                 using var scope = new GraphTransactionScope();
-                ProcessInputNode(graphContext, inputGraphNode, dependency, snapshot, viewProvider, projectPath, ref trackChanges);
+                ProcessInputNode(graphContext, inputGraphNode, dependency, snapshot, viewProvider, projectPath!, ref trackChanges);
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/InputNodeGraphActionHandlerBase.cs]: Let's wait and see if this is a compiler bug or a new restriction on the feature under .NET Framework.

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/InputNodeGraphActionHandlerBase.cs]: Oh wait, looking again I see you haven't updated the compiler in this PR. Are these warnings on `master` already?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/InputNodeGraphActionHandlerBase.cs]: Yes.

**Comment by davkean** [comment]
I would hope actually that they start providing analysis here around unneeded damnit operator.

**Pull request merged**